src/index.html

```diff
@@ -10,13 +10,95 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a href="#section">
```

Comment:
The `href` attribute in the `<a>` tag should not contain spaces. Consider using an anchor link like `#section` without spaces.